src/middleware/verify-and-receive.ts

```diff
@@ -5,7 +5,12 @@ export function verifyAndReceive(
   state: State,
   event: WebhookEvent & { signature: string }
 ): any {
-  const matchesSignature = verify(state.secret, event.payload, event.signature);
+  // verify will validate that the secret is not undefined
+  const matchesSignature = verify(
+    state.secret!,
```

Comment:
I'm typically not a fan of using `!` - ideally I'd have an assertion here about the `state.secret`, but tbh this isn't the worse use of it 🤷

Reply:
We are all still figuring out the whole TypeScript thing, we got much better at it since we migrated `@octokit/webhooks`, but there is surely lots of potential of improvement :)

Reply:
No problem - I'm a big TS fan myself, so happy to help and be pinged on things :)